[THORN-2116] Adding a link to the Community page (#1151)
thorntail/thorntail: docs/index.adoc

include::reference/index.adoc[leveloffset=+2]

# Getting Involved

> Community contributions are always welcome !

Reviewer: ```suggestion
Community contributions are always welcome.
```

> Please visit a https://thorntail.io/community/[Community] page to learn how to get involved.

Reviewer: ```suggestion
For more information about how to get involved, see our https://thorntail.io/community/[Community website].
```
